llvm/lib/CodeGen/RegAllocGreedy.cpp

```diff
@@ -2748,7 +2748,7 @@ bool RAGreedy::runOnMachineFunction(MachineFunction &mf) {
 
   ExtraInfo.emplace();
   EvictAdvisor =
-      getAnalysis<RegAllocEvictionAdvisorAnalysis>().getAdvisor(*MF, *this);
+      getAnalysis<RegAllocEvictionAdvisorAnalysisLegacy>().getProvider()->getAdvisor(*MF, *this);
```

Comment:
Use a temporary variable here to avoid the ugly wrapping clang-format wants here